js/main.js

```diff
@@ -21,7 +21,93 @@ $form.addEventListener('submit', function (event) {
   };
   data.nextEntryId++;
   data.entries.unshift(notes);
-  document.querySelector('img').src = 'images/placeholder-image-square.jpg';
+  document.querySelector('#default-pic').src = 'images/placeholder-image-square.jpg';
   $form.reset();
+  toggleNoEntries();
+  viewSwap('entries');
+  renderEntry(notes);
 }
 );
+
+function renderEntry(entry) {
+  var $eventUnorderList = document.querySelector('ul');
+  var $entryLi = document.createElement('li');
+  var $entryRowDiv = document.createElement('div');
+  $entryRowDiv.setAttribute('class', 'row');
+  $entryLi.appendChild($entryRowDiv);
+  var $entryColHalfImgDiv = document.createElement('div');
+  $entryColHalfImgDiv.setAttribute('class', 'column-half');
+  $entryRowDiv.appendChild($entryColHalfImgDiv);
+  var $entryImg = document.createElement('img');
+  $entryImg.setAttribute('src', entry.link);
+  $entryImg.setAttribute('class', 'picture-frame margin-bot-30');
+  $entryColHalfImgDiv.appendChild($entryImg);
+  var $entryColHalfTextDiv = document.createElement('div');
+  $entryColHalfTextDiv.setAttribute('class', 'column-half');
+  $entryRowDiv.appendChild($entryColHalfTextDiv);
+  var $entryH2Title = document.createElement('h2');
+  var $getTitle = document.createTextNode(entry.title);
+  $entryH2Title.appendChild($getTitle);
+  $entryColHalfTextDiv.appendChild($entryH2Title);
+  var $entryNoteP = document.createElement('p');
+  var $getNote = document.createTextNode(entry.note);
+  $entryNoteP.appendChild($getNote);
+  $entryColHalfTextDiv.appendChild($entryNoteP);
+  $eventUnorderList.appendChild($entryLi);
```

Comment:
You should not be doing this in this function, you are already doing it with the return value where the function is called.